Libraries/Text/Text/RCTTextView.h

```diff
@@ -17,6 +17,9 @@ NS_ASSUME_NONNULL_BEGIN
 - (instancetype)initWithEventDispatcher:(RCTEventDispatcher *)eventDispatcher; // TODO(OSS Candidate ISS#2710739)
 
 @property (nonatomic, assign) BOOL selectable;
+#if TARGET_OS_OSX // TODO(macOS ISS#2323203)
```

Comment:
This is the 'old' tracking number.
Please change to
// TODO(macOS GH#774)

Since this is a multi-line change, also add the beginning ```[``` and ending ```]```